### PR TITLE
DAOS-8590 test: Updates to soak utils needed

### PR DESCRIPTION
**Pull request by mjean308**
Test-tag-hw-large: pr soak_smoke

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

File: src/tests/ftest/soak/soak_utils.py [hunk 1]
```diff
@@ -571,7 +582,11 @@ def launch_server_stop_start(self, pools, name, results, args):
                 self.log.error(
                     "<<<FAILED:dmg system start failed", exc_info=error)
                 status = False
+            for pool in pools:
+                self.dmg_command.pool_query(pool.uuid)
             if status:
+                # Wait ~ 30 sec before issuing the reintegrate
```
Comment (shimizukko) [inline src/tests/ftest/soak/soak_utils.py:588]: Are we waiting 30 sec or 180 sec?

File: src/tests/ftest/soak/soak_utils.py [hunk 2]
```diff
@@ -1068,12 +1083,12 @@ def build_job_script(self, commands, job, nodesperjob):
     # if additional cmds are needed in the batch script
     prepend_cmds = [
         "set -e",
-        "/usr/bin/daos pool query --pool {} ".format(self.pool[1].uuid),
-        "/usr/bin/daos pool query --pool {} ".format(self.pool[0].uuid)
+        "daos pool query --pool {} ".format(self.pool[1].uuid),
```
Comment (shimizukko) [inline src/tests/ftest/soak/soak_utils.py:1086]:
```
[mkano@wolf-32 container]$ daos pool query --help
Usage:
  daos [OPTIONS] pool query [query-OPTIONS] [<pool name or UUID>]

Application Options:
      --debug                    enable debug output
      --verbose                  enable verbose output (when applicable)
  -j, --json                     enable JSON output

Help Options:
  -h, --help                     Show this help message

[query command options]
      -G, --sys-name=            DAOS system name
      -p, --pool=                pool UUID (deprecated; use positional arg)
```
--pool is deprecated. Also might be better to use label.

**Review by shimizukko — DISMISSED** [review]

**Review by dinghwah — DISMISSED** [review]
LGTM

File: src/tests/ftest/soak/soak_utils.py (hunk 1 above)
Comment (mjean308) [reply src/tests/ftest/soak/soak_utils.py:588]: 30 sec; I will change that 180 was a debug change

**Review by shimizukko — APPROVED** [review]

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Pull request merged**